test/ssl-tests/20-cert-select.cnf

```diff
@@ -5,37 +5,37 @@ num_tests = 56
 test-0 = 0-ECDSA CipherString Selection
```

Comment:
This reordering sure makes this a lot harder to figure out what changed in this file..